osu.Game/Database/RealmAccess.cs

```diff
@@ -173,11 +173,6 @@ public RealmAccess(Storage storage, string filename, GameThread? updateThread =
             if (!Filename.EndsWith(realm_extension, StringComparison.Ordinal))
                 Filename += realm_extension;
 
-#if DEBUG
-            if (!DebugUtils.IsNUnitRunning)
-                applyFilenameSchemaSuffix(ref Filename);
-#endif
```

Comment:
What is the purpose of this change?

Reply:
This should be a wrong change. Maybe I accidentally commited it during the test